README.md

```diff
@@ -204,3 +204,32 @@ pop up parachain -f ./tests/zombienet.toml -p https://github.com/r0gue-io/pop-no
 
 > :information_source: Pop CLI will automatically source the necessary polkadot binaries. Currently, these will be built
 > if on a non-linux system.
+
+
+## Testing Pop CLI 
+
+To test the tool locally.
+
+Run the unit tests:
+
+```sh
+cargo test
+```
+
+Run the contracts e2e tests:
+
+```sh
+cargo test --features e2e_contract
+```
+
+Run the parachain e2e tests:
+
+```sh
+cargo test --features e2e_parachain
```

Comment:
No `e2e_parachain` gated test code found.

Reply:
Suggest moving out this unit test:
```rs
        #[test]
	#[ignore = "Build parachain takes long time to build"]
	fn test_parachain_build_after_instantiating_template() -> Result<()> {
		let temp_dir =
			setup_template_and_instantiate().expect("Failed to setup template and instantiate");
		let build = build_parachain(&Some(temp_dir.path().to_path_buf()));
		assert!(build.is_ok(), "Result should be Ok");
		Ok(())
	}
```
 into it's own module `tests/build_parachain.rs` and gating it with `e2e_parachain`